recommendations/test_sms_parser.py

```diff
@@ -1,18 +1,104 @@
 from django.test import TestCase
+from unittest import skip
 from recommendations.sms_parser import SmsParser
 from .actions_list import ACTIONS_LIST
+from faker import Faker
+import random
```

Comment:
@jalletto I used `self.faker.pyint()` instead of `random`. I'm sure either one works, but since we're using faker, I figured we didn't need to import a second library.

Reply:
No action needed, just letting you know why I changed it 😄